cmd/project.go

```diff
@@ -36,22 +36,50 @@ Use the subcommands to manage your projects.`,
 	},
 }
 
-func saveEnv(dir string, apikey string) {
+func saveEnv(dir string, apikey string, projectKey string) {
 	filename := filepath.Join(dir, ".env")
 	envLines, err := env.ParseEnvFile(filename)
 	if err != nil {
 		errsystem.New(errsystem.ErrReadConfigurationFile, err, errsystem.WithContextMessage("Failed to parse .env file")).ShowErrorAndExit()
 	}
-	var found bool
+	var found map[string]bool = map[string]bool{
```

Comment:
```suggestion
	found := map[string]bool{
```

Reply:
also the default value is `false` so you can just do:
```go
found := make(map[string]bool)
```
and your code will still work